project/src/main/java/com/carlease/project/application/ApplicationController.java

```diff
@@ -10,34 +15,43 @@
 @RequestMapping("api/v1/applications")
 public class ApplicationController {
 
+    private final AutosuggestorServiceImpl autosuggestorServiceImpl;
     public ApplicationService applicationService;
 
-    public ApplicationController(ApplicationService applicationService) {
+    public ApplicationController(ApplicationService applicationService, AutosuggestorServiceImpl autosuggestorServiceImpl) {
         this.applicationService = applicationService;
+        this.autosuggestorServiceImpl = autosuggestorServiceImpl;
     }
 
     @GetMapping(produces = "application/json")
-    ResponseEntity<List<Application>> getApplications() {
-        List<Application> list = applicationService.findAll();
+    ResponseEntity<List<ApplicationFormDto>> getApplications() {
+        List<ApplicationFormDto> list = applicationService.findAll();
         return new ResponseEntity<>(list, HttpStatus.OK);
     }
 
     @GetMapping("/{id}")
-    ResponseEntity<Application> getApplication(@PathVariable("id") long id) {
+    ResponseEntity<ApplicationFormDto> getApplication(@PathVariable("id") long id) throws ApplicationNotFoundException {
 
-        Application application = applicationService.findById(id);
-        return new ResponseEntity<>(application, HttpStatus.OK);
+        ApplicationFormDto applicationDto = applicationService.findById(id);
+        return new ResponseEntity<>(applicationDto, HttpStatus.OK);
     }
 
     @GetMapping("/user/{id}")
-    ResponseEntity<List<Application>> getApplicationsByUserId(@PathVariable("user_id") long id) {
-        List<Application> applications = applicationService.findAllByUserId(id);
+    ResponseEntity<List<ApplicationFormDto>> getApplicationsByUserId(@PathVariable("user_id") long id) {
```

Comment:
```suggestion
    ResponseEntity<List<ApplicationFormDto>> getApplicationsByUserId(@PathVariable("userId") long id) {
```
camelCase for path variables